lib/react/rails/railtie.rb

```diff
@@ -38,10 +38,12 @@ class Railtie < ::Rails::Railtie
           addons: app.config.react.addons,
         })
 
-        app.assets.version = [
-          app.assets.version,
-          "react-#{asset_variant.react_build}",
-        ].compact.join('-')
+        if app.assets.nil?
+          app.config.assets.version = [app.config.assets.version, "react-#{asset_variant.react_build}",].compact.join('-')
+        else
+          app.assets.version = [app.assets.version, "react-#{asset_variant.react_build}",].compact.join('-')
```

Comment:
I think this could also be DRYer, eg 

```ruby
sprockets_env = app.assets || app.config.assets # sprockets-rails 3.x attaches this at a different config
sprockets_env.version = [ ... ].compact.join("-")
```